examples/kops-new-cluster/base/cluster.yaml

```diff
@@ -47,16 +47,14 @@ spec:
     anonymousAuth: false
   kubernetesApiAccess:
   - 0.0.0.0/0
-  kubernetesVersion: 1.21.0
   masterInternalName: api.internal.aws-external-cloud-provider-example.k8s.local
   masterPublicName: api.aws-external-cloud-provider-example.k8s.local
   networkCIDR: 172.20.0.0/16
   networking:
-    amazonvpc: {}
+    cilium: {}
```

Comment:
Reasoning?

Reply:
The configuration does not work with amazonvpc because the cluster was expecting an overlay network. This could also have been fixed, but would require more changes.
I just picked cilium (essentially over calico) by random.